### PR TITLE
Adding an html_save method with date option

### PR DESCRIPTION
**Pull request by danvideo**
so we can keep various versions of the html at different times in the sequence

### CODE REVIEW

File: test/mechanize_bot_test.rb [hunk 1]
```diff
@@ -101,6 +101,16 @@ def test_save_html
     File.delete(file_path) if File.exist? file_path
   end
 
+  def test_save_html_w_date
+    file_path = "#{Dir.pwd}/test/test_html-INSERT_DATE.html"
+    @bot.execute_step(:go, @input_test_file)
+    @bot.execute_step(:save_page_html, nil, file_path, nil)
+    new_file_path = Dir::glob("#{Dir.pwd}/test/test_html-*.html").first
```
Comment (virtuexru) [inline test/mechanize_bot_test.rb]: lol THANKS HOUND

File: lib/web_minion/bots/mechanize_bot.rb [hunk 2]
```diff
@@ -39,7 +39,16 @@ def set_file_upload(target, value, element)
...
     end
 
+    # ability to save dynamic dates in the filename with string "INSERT_DATE"
+    # so a value of "myfilename-INSERT_DATE.html" will look like this:
+    # "myfilename-2016-09-19_18-51-40.html"
+    # if you want multiple steps saving at the same time without overwriting,
+    # rename save_html step in your json (i.e. 'save_html_confirmation' etc)
     def save_page_html(_target, value, _element)
+      if value.include?("INSERT_DATE")
+        time = "#{Time.now.strftime('%Y-%m-%d_%H-%M-%S')}"
```
Comment (virtuexru) [inline lib/web_minion/bots/mechanize_bot.rb]: I actually agree w/ this one.. `Time.now.strftime('%Y-%m-%d_%H-%M-%S').to_s`

Comment (virtuexru) [inline lib/web_minion/bots/mechanize_bot.rb]: ahh commenting on github again; so awesome. :(

**Comment by virtuexru** [comment]
:+1:

File: lib/web_minion/bots/mechanize_bot.rb (hunk 2 above)
Comment (stevendaniels) [inline lib/web_minion/bots/mechanize_bot.rb]: strftime is a string anyway.

**Pull request merged**